### PR TITLE
remove spaces

### PR DESCRIPTION
**Pull request by Pavan-04-08**
### Description
acc to https://github.com/Codecademy/docs/pull/1526#issuecomment-1485551055

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@Pavan-04-08 looks good for a second review.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@Pavan-04-08 There is one more change needed.

File: content/go/concepts/functions/functions.md
[the remark this answers is not in the document]
Comment (Pavan-04-08) [reply content/go/concepts/functions/functions.md:1]: hmm there is a new line after 30? @KTom101 
image

[the remark this answers is not in the document]
Comment (Pavan-04-08) [reply content/go/concepts/functions/functions.md:1]: if its refering to 29,30 line in between 29 and 30? 
to combine them i am not actually sure how to place that explaination into 1 line

[the remark this answers is not in the document]
Comment (yangc95) [reply content/go/concepts/functions/functions.md:1]: Yes, it looks like line 30 is now line 29.

In which case, can we add a line between 29+30? In addition, can we rephrase to say that parameters are passed-by-value or passed-by-copy, meaning the function receives...

Also when it says "To modify the parameter, pass a pointer," would you be able to expand on this for me? Do we mean the parameter should be of pointer type? Thanks for any info you can provide

[the remark this answers is not in the document]
Comment (Pavan-04-08) [reply content/go/concepts/functions/functions.md:1]: yeah it should be pointer type , like say

[the remark this answers is not in the document]
Comment (Pavan-04-08) [reply content/go/concepts/functions/functions.md:1]:
```
func main() {
    changeMe := 10
    modify(&x)
    fmt.Println(changeMe) //prints smthng idk whatever modify changes it
}
```

this is kinda similar to any other lang ig not go only thing so i kept it simple

**Review by KTom101 — APPROVED** [review]
@Pavan-04-08 👍

**Pull request merged**